core/src/test/java/org/ehcache/core/internal/service/ServiceLocatorTest.java

```diff
@@ -239,8 +239,7 @@ public void testRedefineDefaultServiceWhileDependingOnIt() throws Exception {
     serviceLocator.startAllServices();
   }
 
-  @Test
-  @Ignore
+  @Test(expected = IllegalStateException.class)
```

Comment:
As you know, I'm not a bit fan of expected exceptions. I prefer a good old try, fail or assertj